### PR TITLE
[12.0][ADD] New module 'sale_product_category_menu': version 12.0

### PR DESCRIPTION
**Pull request by manuelregidor**
This modules adds 'Product Categories' menu item to Sales > Configuration > Products.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please don't use plurals in module names. A better name following conventions is `sale_product_category_menu`.

File: product_categories_in_sales/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Product Categories in Sales",
+    "summary": "Shows 'Product Categories' menu item in Sales",
+    "version": "12.0.1.0.1",
+    "category": "Product",
+    "website": "https://www.sygel.es",
```
Comment (legalsylvain) [inline product_categories_in_sales/__manifest__.py]: should be 'https://github.com/OCA/sale-workflow'

File: product_categories_in_sales/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "author": "Sygel",
```
Comment (legalsylvain) [inline product_categories_in_sales/__manifest__.py]: please Add "Odoo Community Association (OCA)" as a second key

File: product_categories_in_sales/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "sale",
+        "product",
```
Comment (legalsylvain) [inline product_categories_in_sales/__manifest__.py]: I think that the second dependency (product) is unnecessary.

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Hi @manuelregidor. Thanks for this module.

LGTM, except some minor remarks in line + @pedrobaeza remarks, regarding the name of the module.

kind regards.

File: product_categories_in_sales/__manifest__.py (hunk 1 above)
Comment (manuelregidor) [reply product_categories_in_sales/__manifest__.py]: Done.

File: product_categories_in_sales/__manifest__.py (hunk 3 above)
Comment (manuelregidor) [reply product_categories_in_sales/__manifest__.py]: Yes, you're right. Thank you.

**Review by HaraldPanten — APPROVED** [review]
LGTM. 👍 

Do you mind fast tracking @pedrobaeza @legalsylvain @rousseldenis ?

**Review by legalsylvain — APPROVED** [review]
thanks.

File: sale_product_category_menu/views/sale_views.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Manuel Regidor <manuel.regidor@sygel.es>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <menuitem id="sale_product_category" name="Product Categories" action="product.product_category_action_form" parent="sale.prod_config_main" />
+</odoo>
```
Comment (rousseldenis) [inline sale_product_category_menu/views/sale_views.xml]: Please add a blank line

**Review by rousseldenis — APPROVED** [review]
Minor comment.
Code and functional review.

**Comment by rousseldenis** [comment]
Linked to https://github.com/OCA/sale-workflow/issues/1233

File: sale_product_category_menu/views/sale_views.xml (hunk 4 above)
Comment (manuelregidor) [reply sale_product_category_menu/views/sale_views.xml]: Done.

File: product_categories_in_sales/__manifest__.py (hunk 2 above)
Comment (manuelregidor) [reply product_categories_in_sales/__manifest__.py]: Done

**Pull request merged**